core/tps.service.js

```diff
@@ -192,11 +193,11 @@ async function send_user_passwords(owner, from_date, to_date, users, group) {
             '#FROMDATE#':  from.toDateString(),
             '#TODATE#':  to.toDateString(),
             '#EXPIRATION#': CONFIG.tp.extra_expiration,
+            '#LOCKED#': lock.toDateString(),
```

Comment:
Not sure how we want to manage this. As it stand, we will print the locked date everytime (even if its equal to the end date), which might be confusing to the user.

Maybe instead of passing a date directly, optionally passing a full string?
That way, if there is not lock date, there would just be a blank line in the mail

Reply:
Currently, we display the lock date in the TP overview every time, like this:
![image](https://github.com/user-attachments/assets/cafcda2d-f13a-42b1-8457-8729dc74493b)

I don't think it's too bad to display the lock date everywhere. If you think this is important, should we then have "lock" be set to null when the user does not specify a lock date instead of setting it to be equal to the end date?

Reply:
I was mostly talking about the email sent with the passwords:  like [here](https://github.com/genouest/genouestaccountmanager/blob/master/config/default.json.template#L379C6-L379C18).

I'm not sure if adding a line with 'Accounts will be locked on #LOCK' make sense if a user did not select a lock time.

The issue is also that if the user did not select a lock date, it will seems like the accounts will be locked on the end date, and then be deleted 5 days later (the grace period). Might be confusing since they did not ask for a lock.